packages/story-editor/src/components/library/panes/shopping/shoppingPane.js

```diff
@@ -111,46 +129,72 @@ function ShoppingPane(props) {
     return null;
   }
 
-  const dropDownParams = {
-    hasSearch: true,
-    lightMode: true,
-    onChange,
-    getOptionsByQuery: getProductsByQuery,
-    selectedId: product?.productId,
-    dropDownLabel: __('Product', 'web-stories'),
-    placeholder: isLoading ? __('Loading…', 'web-stories') : '',
-    disabled: isLoading ? true : isSaving,
-    primaryOptions: isLoading ? [] : initialOptions,
-    zIndex: 10,
-  };
+  const RenderItemOverride = forwardRef(
+    ({ option, isSelected, ...rest }, ref) => (
+      <StyledListItem
+        ref={ref}
+        width={option.width}
+        active={isSelected}
+        {...rest}
+      >
+        <StyledProductButton>
+          <Button
+            aria-label={__('Add', 'web-stories')}
+            onClick={() => { }}
+            type={BUTTON_TYPES.TERTIARY}
+            size={BUTTON_SIZES.SMALL}
+            variant={BUTTON_VARIANTS.SQUARE}
+          >
+            <Icons.PlusOutline />
+          </Button>
+        </StyledProductButton>
+        <div>
+          <StyledProductImage width={41} src={option?.product?.productImages[0]?.url}></StyledProductImage>
+        </div>
+        <StyledProductDescription>
+          <StyledProductName>
+            <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL} isBold>
+              {option.label}
+            </Text>
+          </StyledProductName>
+          <Text size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL} >
+            {option.product.productPrice}
+          </Text>
+        </StyledProductDescription>
+      </StyledListItem >
+    )
+  );
 
   return (
     <Pane id={paneId} {...props}>
-      <Row>
-        <HelperText>
-          {__(
-            'Select a product to add to the page. It will be displayed as a "dot" on the page, as well as in the page attachment.',
-            'web-stories'
-          )}
-        </HelperText>
-      </Row>
-      <Row>
-        <Datalist.DropDown
-          options={initialOptions}
-          searchResultsLabel={__('Search results', 'web-stories')}
-          aria-label={__('Product', 'web-stories')}
-          {...dropDownParams}
-        />
-      </Row>
-      <Button
-        variant={BUTTON_VARIANTS.RECTANGLE}
-        type={BUTTON_TYPES.SECONDARY}
-        size={BUTTON_SIZES.SMALL}
-        onClick={insertProduct}
-        disabled={!product}
+      <Section
+        data-testid="shapes-library-pane"
+        title={__('Products', 'web-stories')}
       >
-        {__('Insert product', 'web-stories')}
-      </Button>
+        <Row>
+          <HelperText>
+            {__(
+              'This will add products as a tappable dot on your story.',
+              'web-stories'
+            )}
+          </HelperText>
+        </Row>
+        <Row>
+          <Search
```

Comment:
@swissspidy is the `<Search>` component the correct component to be using here?

Here's a screen of where this is at currently (super rough at the moment) -- was going to look into how to remove the "black" background + border for for the container but started thinking maybe this isn't the right component (the dropdown part...) should I just be rendering out list another way?

<img width="490" alt="Screen Shot 2022-04-28 at 5 54 01 PM" src="https://user-images.githubusercontent.com/62242/165854595-13254435-e618-4ce4-a6df-dc00214d704f.png">